src/status_im/data_store/realm/core.cljs

```diff
@@ -51,14 +51,29 @@
   (when realm
     (.close realm)))
 
-(defn migrate-realm [file-name schemas encryption-key]
-  (let [current-version (realm-version file-name encryption-key)]
-    (doseq [schema schemas
-            :when (> (:schemaVersion schema) current-version)
-            :let [migrated-realm (open-realm schema file-name encryption-key)]]
-      (close migrated-realm)))
+(defn reset-realm [file-name schemas encryption-key]
+  (utils/show-popup "Important: Wallet Upgrade" "The Status Wallet will be upgraded in this release. The 12 mnemonic words will generate different addresses and whisper identities (public key). Given that we changed the algorithm used to generate keys and addresses, it will be impossible to re-import accounts created with the old algorithm in Status. Please create a new account.")
```

Comment:
Should this not be called outside of `reset`?

Reply:
not sure, it used to be there I have reintroduced it ( it seems resetting realm is a single purpouse thing, only applies for this release, should be throwaway code), but don't mind putting it outside.

Reply:
We might find other needs for this in the future. I would not expect that kind of side effects when calling a `reset` function.